tests/api-compatibility/acceptable-breakages-v10.0.txt

```diff
@@ -0,0 +1,40 @@
+Compat issues with assembly Mono.Android:
```

Comment:
Instead of `Compat issues with assembly ...`, could we do: `## Assembly: ...`?

e.g.

```
## Assembly: Mono.Android
```